packages/performance/src/utils/errors.ts

```diff
@@ -49,7 +50,9 @@ const ERROR_DESCRIPTION_MAP: { readonly [key in ErrorCode]: string } = {
   [ErrorCode.INVALID_ATTRIBUTE_VALUE]:
     'Attribute value {$attributeValue} is invalid.',
   [ErrorCode.INVALID_CUSTOM_METRIC_NAME]:
-    'Custom metric name {$customMetricName} is invalid'
+    'Custom metric name {$customMetricName} is invalid',
+  [ErrorCode.INVALID_STRING_MERGER_PARAMETER]:
+    'Input for String merger is invalid'
```

Comment:
This is quite a cryptic message for the developers. Can we make this message a bit more contextual? Eg. Unable to form API Key/URL?

Reply:
This message is not planned for developers to resolve by themselves, it is useful to identify the issue quickly when developer takes this message to Firebase customer support.

Reply:
Sure, but still the developer looking at that message would be confused as to how to act on that message.

Reply:
Added more information to let developer know about the next step.